components/virtual-node/virtual-node-rest-service-impl/src/main/kotlin/net/corda/virtualnode/rest/impl/v1/VirtualNodeRestResourceImpl.kt

```diff
@@ -367,20 +370,65 @@ internal class VirtualNodeRestResourceImpl(
         }
     }
 
-    override fun getCreateCryptoSchemaSQL(): ResponseEntity<String> {
-        TODO("To be implemented in CORE-15805")
+    override fun getCreateCryptoSchemaSQL(): String {
+        return getSchemaSql(DbTypes.CRYPTO, null, null)
     }
 
-    override fun getCreateUniquenessSchemaSQL(): ResponseEntity<String> {
-        TODO("To be implemented in CORE-15805")
+    override fun getCreateUniquenessSchemaSQL(): String {
+        return getSchemaSql(DbTypes.UNIQUENESS, null, null)
     }
 
-    override fun getCreateVaultSchemaSQL(cpiChecksum: String): ResponseEntity<String> {
-        TODO("To be implemented in CORE-15805")
+    override fun getCreateVaultSchemaSQL(cpiChecksum: String): String {
+        return getSchemaSql(DbTypes.VAULT, null, cpiChecksum)
     }
 
-    override fun getUpdateSchemaSQL(virtualNodeShortId: String, newCpiChecksum: String): ResponseEntity<String> {
-        TODO("To be implemented in CORE-15805")
+    override fun getUpdateSchemaSQL(virtualNodeShortId: String, newCpiChecksum: String): String {
+        return getSchemaSql(DbTypes.VAULT, virtualNodeShortId, newCpiChecksum)
+    }
+
+    private fun getSchemaSql(
+        dbType: DbTypes,
+        virtualNodeShortId: String?,
+        cpiChecksum: String?
+    ): String {
+        val instant = clock.instant()
+
+        val rpcRequest = VirtualNodeManagementRequest(
```

Comment:
```suggestion
        val managementRequest = VirtualNodeManagementRequest(
```
Not sure if we need `rpc` prefix here.